src/helpers/move-project-card.ts

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2021 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { CardListResponse, PullRequest } from '../types';
+import { getDestinationColumn, getOriginColumn, getProjectColumns } from '../utils/get-project-columns';
+import { context } from '@actions/github';
+import { octokit } from '../octokit';
+
+interface MoveProjectCardProps {
+  project_destination_column_name: string;
+  pull_number: number;
+  project_name: string;
+  project_origin_column_name: string;
+}
+
+export const moveProjectCard = async ({
+  pull_number,
+  project_destination_column_name,
+  project_origin_column_name,
+  project_name
+}: MoveProjectCardProps) => {
+  const getResponse = await octokit.pulls.get({ pull_number, ...context.repo });
+  const pullRequest = getResponse.data as PullRequest;
+  const columnsList = await getProjectColumns({ project_name });
+
+  if (!columnsList) {
+    return null;
+  }
+
+  const destinationColumn = getDestinationColumn(columnsList, project_destination_column_name);
+  const originColumn = getOriginColumn(columnsList, project_origin_column_name);
+
+  if (!originColumn) {
+    return null;
```

Comment:
We could probably just log using `core.info()` that no origin column was found, and just `return`